.gitignore

```diff
@@ -56,3 +56,5 @@ eeprom.bin
 
 # NPM packages folder
 /node_modules
+index_hash.txt
```

Comment:
these changes shouldn't be here, i guess

Reply:
I put it in so my local index_hash doesn't go into the branch/PR but now that change to .gitignore found its way in.
Can you give me a hint on how to deal with the two index files (json and hash) to push them to my branch on github but exclude them from the PR?